BLUESPAWN-client/src/user/BLUESPAWN.cpp

```diff
@@ -5,32 +5,129 @@
 #include "common/DynamicLinker.h"
 #include "common/StringUtils.h"
 #include "util/eventlogs/EventLogs.h"
+#include "hunt/reaction/SuspendProcess.h"
+#include "hunt/reaction/RemoveValue.h"
+
+#include "hunt/hunts/HuntT1004.h"
+#include "hunt/hunts/HuntT1037.h"
+#include "hunt/hunts/HuntT1050.h"
+#include "hunt/hunts/HuntT1055.h"
+#include "hunt/hunts/HuntT1060.h"
+#include "hunt/hunts/HuntT1100.h"
+#include "hunt/hunts/HuntT1101.h"
+#include "hunt/hunts/HuntT1103.h"
+#include "hunt/hunts/HuntT1131.h"
+#include "hunt/hunts/HuntT1138.h"
+#include "hunt/hunts/HuntT1182.h"
+#include "hunt/hunts/HuntT1183.h"
+
+#include "monitor/ETW_Wrapper.h"
+
+#include "mitigation/mitigations/MitigateM1042-LLMNR.h"
+#include "mitigation/mitigations/MitigateM1042-NBT.h"
+#include "mitigation/mitigations/MitigateM1042-WSH.h"
+#include "mitigation/mitigations/MitigateV1093.h"
+#include "mitigation/mitigations/MitigateV1153.h"
+#include "mitigation/mitigations/MitigateV3338.h"
+#include "mitigation/mitigations/MitigateV63597.h"
+#include "mitigation/mitigations/MitigateV63817.h"
+#include "mitigation/mitigations/MitigateV63825.h"
+#include "mitigation/mitigations/MitigateV63829.h"
+#include "mitigation/mitigations/MitigateV72753.h"
+#include "mitigation/mitigations/MitigateV73519.h"
 
 #include <iostream>
 
-int main(int argc, char* argv[])
-{
+IOBase& Bluespawn::io = CLI();
+HuntRegister Bluespawn::huntRecord{ io };
+MitigationRegister Bluespawn::mitigationRecord{ io };
+
+Bluespawn::Bluespawn() {
+
+	huntRecord.RegisterHunt(std::make_shared<Hunts::HuntT1004>());
+	huntRecord.RegisterHunt(std::make_shared<Hunts::HuntT1037>());
+	huntRecord.RegisterHunt(std::make_shared<Hunts::HuntT1050>());
+	huntRecord.RegisterHunt(std::make_shared<Hunts::HuntT1055>());
+	huntRecord.RegisterHunt(std::make_shared<Hunts::HuntT1060>());
+	huntRecord.RegisterHunt(std::make_shared<Hunts::HuntT1100>());
+	huntRecord.RegisterHunt(std::make_shared<Hunts::HuntT1101>());
+	huntRecord.RegisterHunt(std::make_shared<Hunts::HuntT1103>());
+	huntRecord.RegisterHunt(std::make_shared<Hunts::HuntT1131>());
+	huntRecord.RegisterHunt(std::make_shared<Hunts::HuntT1138>());
+	huntRecord.RegisterHunt(std::make_shared<Hunts::HuntT1182>());
+	huntRecord.RegisterHunt(std::make_shared<Hunts::HuntT1183>());
+
+	using namespace Mitigations;
+   
+	MitigateM1042LLMNR* m1042llmnr = new MitigateM1042LLMNR(mitigationRecord);
+	MitigateM1042NBT* m1042nbt = new MitigateM1042NBT(mitigationRecord);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'm1042nbt' is assigned a value that is never used.](https://app.codacy.com/manual/ION28/BLUESPAWN/pullRequest?prid=4967854)